src/app/tests/suites/certification/Test_TC_DRLK_2_12.yaml

```diff
@@ -0,0 +1,435 @@
+      PICS: DRLK.S.F07 && DRLK.S.F00 && DRLK.S.A0033
+      command: "readAttribute"
+      attribute: "RequirePINforRemoteOperation"
+      response:
+          value: false
+
+    - label: "TH sends the Unbolt Door command to the DUT without PINCode"
+      PICS: DRLK.S.C27.Rsp
+      command: "UnboltDoor"
+      timedInteractionTimeoutMs: 1000
+
+    - label: "TH sends the Unbolt Door command to the DUT with valid PINCode"
+      PICS: DRLK.S.C27.Rsp
+      command: "UnboltDoor"
+      timedInteractionTimeoutMs: 1000
+      arguments:
+          values:
+              - name: "PINCode"
+                value: "123456"
+
+    - label:
+          "TH writes the RequirePINforRemoteOperation attribute value as False
+          on the DUT"
+      PICS: DRLK.S.F07 && DRLK.S.F00 && DRLK.S.A0033
+      command: "writeAttribute"
+      attribute: "RequirePINforRemoteOperation"
+      arguments:
+          value: true
+
+    - label:
+          "TH writes the RequirePINforRemoteOperation attribute value as False
+          on the DUT and Verify DUT responds with UNSUPPORTED_WRITE"
+      PICS: DRLK.S.F07 && DRLK.S.F00 && !DRLK.S.A0033
+      command: "writeAttribute"
+      attribute: "RequirePINforRemoteOperation"
+      arguments:
+          value: true
+      response:
+          error: UNSUPPORTED_WRITE
+
+    - label: "TH reads the RequirePINforRemoteOperation attribute from the DUT"
+      PICS: DRLK.S.F07 && DRLK.S.F00 && DRLK.S.A0033
+      command: "readAttribute"
+      attribute: "RequirePINforRemoteOperation"
+      response:
+          value: true
+
+    - label: "TH sends the Unbolt Door command to the DUT with valid PINCode"
+      PICS: DRLK.S.C27.Rsp && DRLK.S.A0033
+      command: "UnboltDoor"
+      timedInteractionTimeoutMs: 1000
+      arguments:
+          values:
+              - name: "PINCode"
+                value: "123456"
+
+    - label: "TH sends the Unbolt Door command to the DUT with invalid PINCode"
+      PICS: DRLK.S.C27.Rsp && DRLK.S.A0033
+      command: "UnboltDoor"
+      timedInteractionTimeoutMs: 1000
+      arguments:
+          values:
+              - name: "PINCode"
+                value: "1234568"
+      response:
+          error: FAILURE
+
+    - label: "TH sends the Unbolt Door command to the DUT without PINCode"
+      PICS: DRLK.S.C27.Rsp && DRLK.S.A0033
+      command: "UnboltDoor"
+      timedInteractionTimeoutMs: 1000
+      response:
+          error: FAILURE
+
+    - label:
+          "TH writes WrongCodeEntryLimit attribute value as 3 on the DUT and
+          Verify that the DUT sends Success response"
+      PICS: DRLK.S.F00 && DRLK.S.F01 && DRLK.S.A0030
+      command: "writeAttribute"
+      attribute: "WrongCodeEntryLimit"
+      arguments:
+          value: 3
+
+    - label:
+          "TH writes WrongCodeEntryLimit attribute value as 3 on the DUT and
+          verify DUT responds with UNSUPPORTED_WRITE"
+      PICS: DRLK.S.F00 && DRLK.S.F01 && ! DRLK.S.A0030
```

Comment:
This does not look like the right PICS check?  The test plan is checking `DRLK.S.A0030.Write`, afaict.  Same in the previous step.